qpmodel/SQLite.g4

```diff
@@ -193,33 +193,27 @@ pred_expr
  * predicate, for example: WHERE case when ... then true ... end
  */
 arith_expr
- : literal_value											#xxLiteralExpr
- | unary_operator arith_expr								#unaryexpr
- | BIND_PARAMETER											#bindexpr
- | ( ( database_name '.' )? table_name '.' )? column_name	#ColExpr
- | signed_number                                            #NumericLiteral
-
-
- | arith_expr op=( '*' | '/' | '%' ) arith_expr				#arithtimesexpr
- | arith_expr op=( PLUS | MINUS ) arith_expr				#arithplusexpr
- | arith_expr op=( '<<' | '>>' | '&' | '|' ) arith_expr		#arithbitexpr
-
- | '(' select_stmt ')'                                      #ScalarSubqueryExpr
- | function_name '(' ( K_DISTINCT? arith_expr ( ',' arith_expr )* | '*' )? ')'	#FuncExpr
-
- | K_CAST '(' arith_expr K_AS type_name ')'					#CastExpr
+ : literal_value											                                                          #xxLiteralExpr
```

Comment:
align labels (#name)

Reply:
The lables are aligned.
![image](https://user-images.githubusercontent.com/4715523/92448609-6f970c00-f1eb-11ea-8181-37aed6c112e5.png)
The file is displayed differently on GitHub.

Reply:
could be the tab/space issue